examples/file_parser.rs

```diff
@@ -27,5 +27,10 @@ fn main() {
         Ok(s) => s,
     };
 
-    assert!(webrtc_sdp::parse_sdp(&s, true).is_ok());
+    let res = webrtc_sdp::parse_sdp(&s, true);
+    if res.is_ok() {
+        println!("successfully parsed SDP.");
+    } else {
+        println!("FAILURE: {}", res.err().unwrap());
```

Comment:
`println!(...)` will output to `standard out`, we probably want that message to end up on `standard error`. Elsewhere we are using `panic!` so we can return the Error result if we want.

```rust
fn main() -> Result<(), webrtc_sdp::SdpParserError> {
...
webrtc_sdp::parse_sdp(&s, true).or_else(|e| { eprintln!("Failed to parse"); Err(e) });
println!("Sucessfully parsed SDP.");
```

Reply:
Thank you. Makes all sense. Addressed.